src/pytest_postgresql/janitor.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         :param version: postgresql version number
         """
         self.user = user
+        self.password = None
```

Comment:
I'd rather use default constructor parameter, rather than additional method. Plus this is not private, so you don't need separate setter for it.